### ISSUE TITLE
Using Parallax scrolling gives React.jsx invalid 

### ISSUE THREAD

**Issue by luveshsadhwani** [pmndrs/react-spring#1538]
## 🐛 Bug Report

Hi, I am trying to implement Parallax scrolling using this library and I got an error saying React.jsx type is invalid. Upon importing parallax from the core module, I logged the element and realized it had no functionality Upon further inspection, i tried to do a search for Parallax on VS code and could not find any reference to folders under node modules, only babel loader (i ensured I can search for files in node modules. I compared this by searching for Spring & Animation and i did see references to files from node modules. May I know how I can fix this or which version would be best for using Parallax scrolling?

## Environment

- `react-spring` v9.0.0-rc.3
- `react` v17.0.2

**Comment by joshuaellis** [comment]
Can you update spring to the latest version please? We're unable to support release candidates.

**Comment by luveshsadhwani** [comment]
Hey Joshua, thanks for the prompt response. I've updated to  9.2.1 and I still have the same issue

**Comment by joshuaellis** [comment]
Then I ask you follow the bug template and provide a repo of the issue as we haven't had this report and our demos work with Parallax.

What's the core module?

You should do: `import { Parallax, ParallaxLayer, } from '@react-spring/parallax'`

**Comment by luveshsadhwani** [comment]
Hi, sorry i'm new to this. How do i provide a repo of the issue?

Also i tried importing from '@react-spring/parallax' and i got the following error.

![image](https://user-images.githubusercontent.com/71253813/120674889-226c7000-c4c7-11eb-9822-4ed8ec0ce3b0.png)

I changed it to '@react-spring/web' and this removes the error (I used web as it was suggested to do so in the docs). I created the Parallax element and saw this:

![image](https://user-images.githubusercontent.com/71253813/120675259-7d9e6280-c4c7-11eb-9388-ad8685efa539.png)
